devices/Bmm150/Bmm150.nfproj

```diff
@@ -0,0 +1,81 @@
+    <ProjectGuid>{8F39A428-925A-4788-B263-B95761334BCE}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <FileAlignment>512</FileAlignment>
+    <RootNamespace>Iot.Device.Bmm150</RootNamespace>
+    <AssemblyName>Iot.Device.Bmm150</AssemblyName>
+    <TargetFrameworkVersion>v1.0</TargetFrameworkVersion>
+    <DocumentationFile>bin\$(Configuration)\Iot.Device.Bmm150.xml</DocumentationFile>
+    <LangVersion>9.0</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <Import Project="$(NanoFrameworkProjectSystemPath)NFProjectSystem.props" Condition="Exists('$(NanoFrameworkProjectSystemPath)NFProjectSystem.props')" />
+  <ItemGroup>
+    <Reference Include="mscorlib, Version=1.10.5.4, Culture=neutral, PublicKeyToken=c07d481e9758c731">
+      <HintPath>packages\nanoFramework.CoreLibrary.1.10.5\lib\mscorlib.dll</HintPath>
+      <Private>True</Private>
+      <SpecificVersion>True</SpecificVersion>
+    </Reference>
+    <Reference Include="nanoFramework.Hardware.Esp32, Version=1.3.3.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
+      <HintPath>packages\nanoFramework.Hardware.Esp32.1.3.3-preview.22\lib\nanoFramework.Hardware.Esp32.dll</HintPath>
+      <Private>True</Private>
+      <SpecificVersion>True</SpecificVersion>
+    </Reference>
+    <Reference Include="nanoFramework.Runtime.Events, Version=1.9.1.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
+      <HintPath>packages\nanoFramework.Runtime.Events.1.9.1-preview.7\lib\nanoFramework.Runtime.Events.dll</HintPath>
+      <Private>True</Private>
+      <SpecificVersion>True</SpecificVersion>
+    </Reference>
+    <Reference Include="System.Device.I2c, Version=1.0.1.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
+      <HintPath>packages\nanoFramework.System.Device.I2c.1.0.1-preview.39\lib\System.Device.I2c.dll</HintPath>
+      <Private>True</Private>
+      <SpecificVersion>True</SpecificVersion>
+    </Reference>
+    <Reference Include="System.Math, Version=1.4.1.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
+      <HintPath>packages\nanoFramework.System.Math.1.4.1-preview.10\lib\System.Math.dll</HintPath>
+      <Private>True</Private>
+      <SpecificVersion>True</SpecificVersion>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="app.config" />
+    <None Include="packages.config" />
+    <None Include="README.md" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Bmm150.cs" />
+    <Compile Include="Bmm150Compensation.cs" />
+    <Compile Include="Bmm150I2c.cs" />
+    <Compile Include="Bmm150I2cBase.cs" />
+    <Compile Include="Bmm150TrimRegister.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Register.cs" />
+    <None Include="*.md" />
+  </ItemGroup>
+  <Import Project="..\..\src\BinaryPrimitives\BinaryPrimitives.projitems" Label="Shared" />
```

Comment:
As we've recently been creating nuget for all of those, can you please move to the nuget. To the question: but it's project reference with the others, yes, mainly because of time to adjust for nuget!

Reply:
Removed. We do not make use of it for Bmm150.

Reply:
Please move the project reference to nuget. They are now available as nuget. I know the sample you use to create this one had project reference. As mentioned, it's because no time yet to address. So for this new one, use the nuget.
And please don't forget to add them to the nuget dependency list.

Reply:
And please once you'll adjust all that, make sure the nfproj is clean and the shared projects do not appear here.
You'll have as well to adjust the sample to add them as nugets.

Reply:
Done! Removed shared projects, replaced with latest stable nugets.

Reply:
you still have shared projects. Please replace them by nugets. And remove them from here. And add them in the nuspec.